src/core/agent/records/identifierStorage.ts

```diff
@@ -95,9 +95,12 @@ class IdentifierStorage {
     return null;
   }
 
-  async getMultisigMemebersMetadata(): Promise<IdentifierMetadataRecord[]> {
+  async getMultisigMembersMetadata(
+    id: string
+  ): Promise<IdentifierMetadataRecord[]> {
```

Comment:
Hmm, if we are passing an ID, can we not just use `findById` and check that `groupCreated` is true after fetching?

Reply:
Actually, I do not use this in the multisigService anymore.

Reply:
OK but `id` should be removed from the tags then